engine/flink/process/src/test/scala/pl/touk/nussknacker/engine/process/helpers/SampleNodes.scala

```diff
@@ -480,6 +484,88 @@ object SampleNodes {
 
   }
 
+  object GenericParametersSource extends FlinkSourceFactory[AnyRef] with SingleInputGenericNodeTransformation[Source[AnyRef]] {
+
+    override type State = Nothing
+
+    override def contextTransformation(context: ValidationContext, dependencies: List[NodeDependencyValue])(implicit nodeId: NodeId)
+      : this.NodeTransformationDefinition = {
+      case TransformationStep(Nil, _) => NextParameters(initialParameters)
+      case TransformationStep(("type", DefinedEagerParameter(value: String, _))::Nil, None) =>
+        val versions = value match {
+          case "type1" => List(1, 2)
+          case "type2" => List(3, 4)
+          case _ => ???
```

Comment:
Please add comment that this is only for example.

Reply:
done